roles/contiv_network/templates/aci_gw.j2

```diff
@@ -19,7 +19,7 @@ start)
     -e "APIC_EPG_BRIDGE_DOMAIN={{ apic_epg_bridge_domain }}" \
     -e "APIC_CONTRACTS_UNRESTRICTED_MODE={{ apic_contracts_unrestricted_mode }}" \
     --name=contiv-aci-gw \
-    contiv/aci-gw
+    {{ aci_gw_version }}
```

Comment:
yes, variables are the way to do this and you set their defaults in the defaults dir. For `contiv_network` it will be in https://github.com/contiv/ansible/blob/master/roles/contiv_network/defaults/main.yml

Also if you want the use to specify a container image and not just version, please consider renaming `aci_gw_version` to `aci_gw_image`, though I feel we should keep image name and version separate.